services/skus/service_nonint_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"time"
 
 	"github.com/awa/go-iap/appstore"
+	"github.com/brave-intl/bat-go/services/skus/radom"
```

Comment:
nit: This import should go to the last group, this is a place for 3rd-party imports.